### ISSUE TITLE
The setting is not forced if two identical commands with manual change in the meantime

### ISSUE THREAD

**Issue by JardaG** [marex/reserial#1]
I may have encountered one problem - it does not respect/force the execution of the same command again if it has been switched manually in the meantime.
Is it caused by dedicated serial realy controller?

**Comment by JardaG** [comment]
Even in spite of the mask set in the command, it toggles manually set on/off states directly on the socket.

**Comment by marex** [comment]
You're right, I see that too. It's likely some yet-undecoded part of the serial protocol between the MIPS and the STM chip controlling the sockets. Do you have a chance to take a look into this ?

**Comment by JardaG** [comment]
Maybe, but how? I have Saleae Logic Pro 16 available, so I can see what's going on inside, but I'm not sure I'll be able to find out anything from the captured data.

**Comment by marex** [comment]
There is a 3V3 TTL UART serial connection between the MIPS and STM chip inside the device, so you could use USB-to-TTL-UART adapter to connect to the STM chip and debug it on a PC (note that there is 240V inside the device, be very careful). The other option which might work would be to use strace or LD_PRELOAD on the binary from the device to figure out what it does in such situation.
